message_printer.go

```diff
@@ -0,0 +1,105 @@
+package pterm
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pterm/pterm/internal"
+)
+
+var (
+	// MessageVanilla returns a default MessagePrinter, which can be used to print text as is
+	// No default style is present for MessagePrinter
+	MessageVanilla = MessagePrinter{
+		Style: nil,
+	}
+)
+
+// MessagePrinter is the printer used to print the input as-is or as specified by user formatting
+type MessagePrinter struct {
+	Style Style
+	Fatal bool
+}
+
+// WithStyle adds a style to the printer.
+// unlike prefix printer, there is a single style for the message=
+func (p MessagePrinter) WithStyle(style Style) *MessagePrinter {
+	p.Style = style
+	return &p
+}
+
+// WithFatal sets if the printer should panic after printing.
+// NOTE:
+// The printer will only panic if either MessagePrinter.Println, MessagePrinter.Print
+// or MessagePrinter.Printf is called.
+func (p MessagePrinter) WithFatal(b ...bool) *MessagePrinter {
+	p.Fatal = internal.WithBoolean(b)
+	return &p
+}
+
+// Sprint formats using the default formats for its operands and returns the resulting string.
+// Spaces are added between operands when neither is a string.
+func (p MessagePrinter) Sprint(a ...interface{}) string {
+	if p.Style == nil {
+		return fmt.Sprint(a...)
+	}
+	return p.Style.Sprint(a...)
```

Comment:
This is right, but you don't need the Style nil check. If the Style is nil, it will automatically print with normal text.